ui/src/settings.vue

```diff
@@ -342,6 +397,9 @@ h5 {
     margin-bottom: 20px;
     opacity: 0.7;
 }
+.Password {
+    max-width: 100% !important;
```

Comment:
Can we avoid using `!important` by increasing the selector specificity? Do we really need it?